worker/src/main/resources/run.sh

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
-
-SOURCE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-DEPS_DIR="$SOURCE_DIR/../../../target/staged"
-QUARTZ_PROPS_FILE="$SOURCE_DIR/quartz.properties"
-
-java -Dorg.quartz.properties=$QUARTZ_PROPS_FILE -Dpidfile.path=/tmp/pid -Dnetworkaddress.cache.ttl=10 -Dnetworkaddress.cache.negative.ttl=10 -cp "${DEPS_DIR}/*" com.lucidchart.piezo.Worker
```

Comment:
Apache, Jenkins, Nginx, IDEA, etc.

Writing your own init or run scripts is generally not the preferred way.

The common way is to have them work mostly "out of the box", with the ability to drop configuration in a separate file to override defaults.

I added that in `https://github.com/pauldraper/piezo/blob/pauldraper-refactor-build/admin/templates/conf`, which is placed in `/etc/piezo-admin/conf`.